project/new_project_button.js

```diff
@@ -16,20 +16,16 @@ class NewProjectButton extends HTMLElement {
 			this.outerHTML = "";
 		}
 
-		const image = document.createElement("img");
-		image.src = this.getAttribute("src");
+		const span = document.createElement("span");
+		span.innerText = "New Project";
 
 		const anchor = document.createElement("a");
 		anchor.href = this.getAttribute("href");
 		anchor.onclick = this.clear_session;
-		anchor.appendChild(image);
```

Comment:
Previously the anchor was attached to the image, not the span (spans are more decoratable text objects in HTML). When I did the CSS updates to go to just a text view, this got left behind.  Fast fix.